.github/workflows/ci.yml

```diff
@@ -5,7 +5,6 @@ on:
     branches: 
       - 'main'
     paths-ignore:
-      - 'docs/**'
```

Comment:
Overall this still LGTM, but is there any concern around building things that haven't changed as part of PR checks? Seems like the build is only 10-15min as of now, even after adding the docs phase, but being unaware of the economics I thought I'd call it out 🙂 (Do you have to pay for "VM core minutes" or anything like that?)